docs/antctl.md

```diff
@@ -679,3 +681,33 @@ worker1 172.18.0.4 Alive
 worker2 172.18.0.3 Alive 
 worker3 172.18.0.2 Dead
 ```
+
+### Upgrade existing objects of CRDs
+
+antctl supports upgrading existing objects of Antrea CRDs to the latest version.
+The related sub-commands should be run out-of-cluster.
```

Comment:
We should mention the required permissions. It includes the get, list, and update of each CRD and its objects.
And make sure proper error is printed when some permissions are missing.

Reply:
Will add.

Reply:
Running outside the cluster doesn't mean the kubeconfig file has admin priviledge. Normally we should add the required permissions of this command to ClusterRole antctl as the existence of the ClusterRole implies it contains all permissions its commands require. However, the required permissions of `upgrade api-storage` could make antctl a super user, while its privilege is currently quite limited, the only create permission it get is creating supportbundles.

So a conservative approach is to document clearly what permissions are required for the upgrade command, then users know what they need to do when they meet encounter issue.

I don't know how `antctl mc` guides users to get required permissions. But for other commands, we have added all permissions to ClusterRole antctl. For `upgrage api-storage`, we need at least document it.

cc @jianjuns @antoninbas

Reply:
Got that, I'll add the required permissions in doc and give some error prints for lacking permissions.

Reply:
I think there is no document or permissions added to ClusterRole when we add the `antctl mc`, I will create an issue to track this first.

Reply:
Done